### PR TITLE
improve enum page layout

### PR DESCRIPTION
**Pull request by superyyrrzz**
@chenkennt @vwxyzh @ansyral @hellosnow @qinezh 

refer to new enum layout in docs (e.g. https://docs.microsoft.com/en-us/dotnet/core/api/system.uriformat#System_UriFormat), adjust enum page layout
old layout: http://docascode.github.io/docfx-seed/api/MRef.Demo.Enumeration.ColorType.html
new layout: http://superyyrrzz.github.io/SEED/api/MRef.Demo.Enumeration.ColorType.html#MRef_Demo_Enumeration_ColorType

### CODE REVIEW

File: src/docfx.website.themes/default/ManagedReference.html.primary.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+var mrefCommon = require('./ManagedReference.common.js');
+
+exports.transform = function (model) {
+  model = mrefCommon.transform(model);
+  if (model.type.toLowerCase() == "enum") {
```
Comment (hellosnow) [inline src/docfx.website.themes/default/ManagedReference.html.primary.js]: beter ===?

File: src/docfx.website.themes/default/partials/enum.tmpl.partial [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+<h5 id="{{id}}_remarks"><strong>{{__global.remarks}}</strong></h5>
+<div class="markdown level0 remarks">{{{remarks}}}</div>
+{{/remarks}}
+{{#example.0}}
+<h5 id="{{id}}_examples"><strong>{{__global.examples}}</strong></h5>
+{{/example.0}}
+{{#example}}
+{{{.}}}
+{{/example}}
+{{#children}}
```
Comment (hellosnow) [inline src/docfx.website.themes/default/partials/enum.tmpl.partial]: this line before is same with class template, not sure need we extract to some common partial template?

**Comment by vwxyzh** [comment]
New layout looks great!💎

**Comment by hellosnow** [comment]
:shipit:

**Comment by qinezh** [comment]
:shipit:

**Pull request merged**